### DISCUSSION TITLE
Configurations for DC network

### DISCUSSION THREAD

**Discussion by AimanKhan1997** [zach401/acnportal#97] [Q&A] [answered]
Hi, I am working on DCFC model. I had a query regarding [1.2 infrastructure comparison](https://github.com/caltech-netlab/acnportal-experiments/blob/master/examples/1-Infrastructure-Evaluation/1.2-Comparing-Infrastructure-Designs/1.2-comparing-infrastructure-designs.ipynb) in examples.
In the experimental parameters, we define the period, the voltage and DEFAULT_BATTERY_POWER. If I want to compare the level 3 infrastructure along with level 1 and level 2 chargers, do I have to change the experimental parameters. For example keeping the value of voltage to 600 V and the DEFAULT_BATTERY_POWER to 60 KW for DC. If so, will it not effect the the level 1 and level 2 chargers as they require an onboard charger of 6.6 KW.
How should I proceed?

**Comment by sunash** [comment, marked as answer]
Changing the VOLTAGE and DEFAULT_BATTERY_POWER parameters should be fine, as those are only used to determine EV battery parameters for each event in the network in example 1.2. You'll need to edit the voltage parameter for the relevant EVSEs in the network, though. @zach401 could probably weigh in on the physical effects of such a change better than I can, though.

One question, are you trying to re-run 1.2 with DCFC charging? If so, there are some additional things to consider; namely, that events in that experiment are generated from level-2 charging data, and so applying them to level-3 chargers may not be well-principled (since average power demand on a level 3 charger is typically much higher than those on a level 2 charger in practice).

**Reply by AimanKhan1997** [reply]
Thanks for the answer. I had this doubt as DCFC supplies power directly to battery while level 2 supplies power to an onboard charger (in this case of 6.6 KW) and then to the battery. I just wanted to know if simply changing the value can simulate this change in process. 

Yes, initially I did but faced this issue that you just mentioned. Please let me know if you have any suggestion on how to solve this. 

Moreover, in your example [https://github.com/caltech-netlab/gym-acnportal/blob/master/tutorials/lessonx_training_running_rl_agent.ipynb] will I face the same issue if I just change the value of max_rate and voltage in random_plugin function to 125 A and 600V?

**Reply by zach401** [reply]
Yes, increasing the DEFAULT_BATTERY_POWER of the battery will model bypassing the vehicle's onboard BMS. What value is reasonable will depend on the EV you want to model. 

As for increasing the voltage of the EVSE that would be ok, however you would want to use voltage on the AC side, not DC. Most DCFCs are connected at 277 V line-to-neutral (480 line-to-line) in the US. I am not sure what the voltage norms are in Europe or elsewhere.
